e2e/util.go

```diff
@@ -99,7 +99,7 @@ func checkAndK8sPortForwardRedis(ctx context.Context, clientset *kubernetes.Clie
 		return nil, fmt.Errorf("no master pod found")
 	}
 
-	fw, err := portForward(ctx, clientset, config, master, stopChan, resources.DragonflyAdminPort)
+	fw, err := portForward(ctx, clientset, config, master, stopChan, resources.DragonflyPort)
```

Comment:
I see `portForward` doesn't use the last parameter (port). So there is no impact of this line change. Maybe update the portForward definition?